applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -8,6 +8,7 @@ Device Firmware Upgrade MCUmgr module
    :depth: 2
 
 The Device Firmware Upgrade MCUmgr module is responsible for performing a Device Firmware Upgrade (DFU) over Simple Management Protocol (SMP).
+Module is supporting usage of either mcuboot bootloader or SUIT tool for preforming DFU.
```

Comment:
1. MCUboot*
2.  I do not recall seeing SUIT being called "tool" in other docs. IMO it will be better to just leave it named "SUIT".

Reply:
Also this is first mention in this doc, consider explaining what the SUIT is: `Software Updates for Internet of Things (SUIT)`